lib/units/test/unit_test.cpp

```diff
@@ -238,7 +247,7 @@ TEST(UnitParseTest, singular_plural) {
 TEST(UnitFormatTest, roundtrip_string) {
   for (const auto &s :
        {"m", "m/s", "meV", "pAh", "mAh", "ns", "counts", "counts^2",
-        "counts/meV", "1/counts", "counts/m", "Y", "M", "D"}) {
+        "counts/meV", "1/counts", "counts/m", "rad", "$", "Y", "M", "D"}) {
```

Comment:
Why is `$`?

Reply:
For the 'currency' base. LLNL/Units represents all currencies as `$`

Reply:
I didn't even realize there *was* a currency base :D